RxCocoa/Foundation/URLSession+Rx.swift

```diff
@@ -184,12 +184,12 @@ extension Reactive where Base: URLSession {
     - returns: Observable sequence of response data.
     */
     public func data(request: URLRequest) -> Observable<Data> {
-        return response(request: request).map { (response, data) -> Data in
-            if 200 ..< 300 ~= response.statusCode {
-                return data
+        return response(request: request).map { pair -> Data in
```

Comment:
You might want to wait with that since [changes to tuple will be rolled back](https://lists.swift.org/pipermail/swift-evolution-announce/2017-June/000386.html).